ros1_bridge/__init__.py

```diff
@@ -117,6 +121,20 @@ def get_ros1_messages(rospack=None):
 
 def get_ros2_messages():
     msgs = []
+    rules = []
+    rules.append(MappingRule({
+        'ros1_package_name': 'nav_msgs',
+        'ros1_message_name': 'GridCells',
+        'ros2_package_name': 'nav_msgs',
+        'ros2_message_name': 'GridCells',
+        'fields_1_to_2': {
+            'header': 'header',
+            'cell_width': 'cell_width_new',
```

Comment:
Is this `_new` suffix intentional? This doesn't look like an automated test to me.